### PR TITLE
Map Matrix3f and Matrix4f

### PR DESCRIPTION
**Pull request by enbrain**

### CODE REVIEW

**Review by Gegy — APPROVED** [review]

File: mappings/net/minecraft/util/math/Matrix4f.mapping [hunk 1]
```diff
@@ -48,13 +54,36 @@ CLASS net/minecraft/class_1159 net/minecraft/util/math/Matrix4f
...
+	METHOD method_35434 load (Lnet/minecraft/class_1159;)V
+		ARG 1 source
+	METHOD method_35435 read (Ljava/nio/FloatBuffer;)V
+		ARG 1 buf
+	METHOD method_35436 read (Ljava/nio/FloatBuffer;Z)V
+		ARG 1 buf
+		ARG 2 transposed
+	METHOD method_35437 isInteger (F)Z
+		ARG 0 value
+	METHOD method_35438 readTransposed (Ljava/nio/FloatBuffer;)V
```
Comment (liach) [inline mappings/net/minecraft/util/math/Matrix4f.mapping]: Imo we should use row first/column first than transpose or not for the reading order.

Comment (enbrain) [reply mappings/net/minecraft/util/math/Matrix4f.mapping]: Just wondering, does unpick support boolean parameters? like `MatrixFormat.ROW_FIRST` / `MatrixFormat.COLUMN_FIRST`.

Comment (liach) [reply mappings/net/minecraft/util/math/Matrix4f.mapping]: Not yet. If it does, it will be in the `LiteralType` class of unpick.

**Review by liach — APPROVED** [review]
:heart:

**Review by YanisBft — APPROVED** [review]

**Pull request merged**